insights/tests/datasources/test_eap_reports.py

```diff
@@ -0,0 +1,59 @@
+import os
+import pytest
+import shutil
+import tempfile
+from datetime import datetime, timedelta
+
+from insights.core import dr
+from insights.core.context import HostContext
+from insights.specs.datasources.eap_reports import eap_report_files
+
+
+@pytest.fixture
+def sample_directory(scope="module"):
```

Comment:
Instead of creating such real files for testing, and then removing them by hand, the `unittest.mock` may help for such test.